XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/OpenVR/OpenVRDataProvider.cs

```diff
@@ -113,7 +116,7 @@ protected override GenericJoystickController GetOrAddController(string joystickN
         /// <inheritdoc />
         protected override SupportedControllerType GetCurrentControllerType(string joystickName)
         {
-            if (string.IsNullOrEmpty(joystickName) || !joystickName.Contains("OpenVR"))
+            if (string.IsNullOrEmpty(joystickName) || joystickName.Contains("<0"))
```

Comment:
interesting. what is `<0`?

Reply:
As discussed in Slack, this is the extra joystick that Unity generates when you press the Back button. ```
<0xcb29bc9c>
```
So this is to ensure it's excluded.  If we get more from other platforms, I'll turn it into a blacklist.  For now, it's any joystick which looks like an Escape sequence.